### DISCUSSION TITLE
新人绳斤来了

### DISCUSSION THREAD

**Discussion by liaomaya** [share121/inter-knot#5379] [绝区零]
大家好，我是liaoma欢迎大家来本帖聊天
![Uploading Image_1727062642863.jpg…]()

**Comment by KELERSAMA** [comment]
可以看看
#4990   搜索词：绳网常见问题的解决方法 
#4628   搜索词：绳网社区规则详情与准则 
#1436   搜索词：如何让你的帖子更加与众不同 
#4639   搜索词：在github使用米游社表情包
#2106     顺带辟谣一下
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by Yizakl** [reply]
大佬
